From 64fa557eb30fc1219bec50a45107ea1a983411ed Mon Sep 17 00:00:00 2001
From: Sam James <sam@gentoo.org>
Date: Sat, 18 Jun 2022 17:11:11 +0100
Subject: [PATCH] Fix build with libcxx (missing array include)

Bug: https://bugs.gentoo.org/833488

Task-number: QTBUG-104435
Change-Id: I06384761a5560b81b675e6c4ae498bb93dcb4f4f
Pick-to: 5.15 6.2 6.3 6.4
Reviewed-by: Marc Mutz <marc.mutz@qt.io>
(cherry picked from commit 5065013b0c2346b5918a2681ae2e58046140e8a7)
---
 .../compositor/linux-dmabuf-unstable-v1/linuxdmabuf.h           | 2 ++
 1 file changed, 2 insertions(+)

--- a/src/hardwareintegration/compositor/linux-dmabuf-unstable-v1/linuxdmabuf.h
+++ b/src/hardwareintegration/compositor/linux-dmabuf-unstable-v1/linuxdmabuf.h
@@ -41,6 +41,8 @@
 #include <QtCore/QTextStream>
 #include <QtGui/QOpenGLTexture>
 
+#include <array>
+
 #include <EGL/egl.h>
 #include <EGL/eglext.h>
 
